app/controllers/api_controller/parameters.rb

```diff
@@ -132,7 +132,7 @@ def sort_params(klass)
       options = String(params['sort_options']).split(",")
       params['sort_by'].split(",").zip(orders).collect do |attr, order|
         raise BadRequestError,
-              "#{attr} is not a valid attribute for #{klass.name}" if !klass.respond_to?(attr) && attr != "id"
+              "#{attr} is not a valid attribute for #{klass.name}" if !klass.method_defined?(attr) && attr != "id"
```

Comment:
:+1: I was wondering how the other sorting specs could have been passing before, but I see they were sorting by name, which every class responds to :laughing: